scripts/tasks/jest.ts

```diff
@@ -1,22 +1,44 @@
-import { jestTask, argv } from 'just-scripts';
+import { jestTask, argv, JestTaskOptions } from 'just-scripts';
 import * as path from 'path';
 
-const commonArgs = () => {
+/**
+ * Partial support of native jest CLI arguments https://jestjs.io/docs/en/cli
+ * > Why partial support only? just jestTask limits support to only those specified as return value of this function
+ */
+const commonArgs = (): JestTaskOptions => {
+  const args: JestTaskOptions = argv();
+
   return {
-    ...((process.env.TF_BUILD || process.env.LAGE_PACKAGE_NAME) && { runInBand: true }),
-    ...(argv().u || argv().updateSnapshot ? { updateSnapshot: true } : undefined),
+    ...((process.env.TF_BUILD || process.env.LAGE_PACKAGE_NAME || args.runInBand) && { runInBand: true }),
+    ...((args.u || args.updateSnapshot) && { updateSnapshot: true }),
+    clearCache: args.clearCache,
+    config: args.config,
+    watch: args.watch,
+    coverage: args.coverage,
+    passWithNoTests: args.coverage,
+    testNamePattern: args.testNamePattern,
+    testPathPattern: args.testPathPattern,
```

Comment:
I think we need to add back support for passing through extra args; otherwise it won't be possible to pass a test name.
```suggestion
    testPathPattern: args.testPathPattern,
    _: (args._ || []).filter(arg => arg !== 'jest' && arg !== 'jest-watch'),
```

Reply:
that's why we have now standard `--testNamePattern` and `--testPathPattern`, but you're right, this would be a breaking change.